wireshark/README.md

```diff
@@ -19,6 +19,9 @@
 
 -->
 
+Note: this lua script may cause crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). 
+Please make sure the version of Wireshark is lower than v4.0.0.
```

Comment:
```suggestion
**Note**: this Lua script may cause a crash in the newest version of Wireshark, see https://github.com/apache/pulsar/issues/18439.  Please make sure the version of Wireshark is lower than v4.0.0.
```

Make use of GitHub Markdown flavor.

Reply:
Fine. Then I'll remove the links magic but retain the bold trick as there's no breaking regression:

```suggestion
> **Note**:this Lua script may cause a crash in the newest version of Wireshark, see [#18439](https://github.com/apache/pulsar/issues/18439). Please make sure the version of Wireshark is lower than v4.0.0.
```